Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
-name = "jsontofile"
-version = "0.1.0"
+name = "jsonsplitter"
```

Comment:
i didn't like this name 
can you think of a better name

Reply:
Sorry, I can't think of any better name.

Reply:
ok i understand